pkg/util/general/common.go

```diff
@@ -362,3 +362,19 @@ func FormatMemoryQuantity(q float64) string {
 
 	return fmt.Sprintf("%v[%v]", q, quantity.String())
 }
+
+// DedupStringSlice return deduplicated string slice from original
+func DedupStringSlice(input []string) []string {
```

Comment:
this DedupStringSlice can be also implemented by sets.String and return UnsortedList

Reply:
Got it